src/api/app/views/webui2/webui/interconnects/new.html.haml

```diff
@@ -1,6 +1,40 @@
 :ruby
   @pagetitle = 'Interconnect'
+  opensuse_exists = Project.exists?(name: 'openSUSE.org')
+  tizen_exists = Project.exists?(name: 'tizen.org')
+  packman_exists = Project.exists?(name: 'Packman')
 
 .card.mb-3
   = render partial: 'tabs'
   .card-body
+    %h3 Connect a remote Open Build Service instance
+    -# %span Use a public OBS for default base distributions
+    .list-group.my-3
+      = render partial: 'interconnect_button', locals: { name: 'openSUSE.org',
+                                                         remoteurl: 'https://api.opensuse.org/public',
+                                                         title: 'Standard OBS instance at build.opensuse.org',
+                                                         description: 'This instance delivers the default build targets for OBS.',
+                                                         type: 'opensuse',
+                                                         exists: opensuse_exists }
+      = render partial: 'interconnect_button', locals: { name: 'tizen.org',
+                                                         remoteurl: 'https://api.tizen.org/public',
+                                                         title: 'Standard OBS instance at build.opensuse.org',
+                                                         description: 'This instance delivers the default build targets for OBS.',
+                                                         type: 'tizen',
+                                                         exists: tizen_exists }
+      = render partial: 'interconnect_button', locals: { name: 'Packman',
+                                                         remoteurl: 'https://pmbs-api.links2linux.de/public',
+                                                         title: 'Packman Build Service PMBS',
+                                                         description: 'This instance can be used to access resources from packman.',
+                                                         type: 'packman',
+                                                         exists: packman_exists }
+      .list-group-item
+        .d-flex.w-100.justify-content-between
+          %h5.mb-1 Custom OBS instance
+          = button_tag 'Add', class: 'add_interconnect', data: { toggle: 'modal', target: '#interconnect-modal' }
+        %small.text-muted This project is representing a remote build service instance.
+
+= render partial: 'interconnect_modal'
+
+:javascript
```

Comment:
This should be in the ready function:

```
= content_for :ready_function do
  initInterconnect();
```

Reply:
why? it is a `$(document).on('ajax:success')`

Reply:
You are right, my bad!